frontend/src/component/JobPostingCard.tsx

```diff
@@ -0,0 +1,35 @@
+import {JobPosting} from "../model/JobPosting";
+import {Button, Card, CardActions, CardContent, Typography} from "@mui/material";
+
+export type JobPostingCardProps = {
+    jobPosting: JobPosting
```

Comment:
Tipp: füge nach dem letzten Element ein `,` oder ein `;` ein, damit bei neuen zusätzlichen Elementen in Zukunft die Git-Historie einfacher zu verstehen ist.
```suggestion
    jobPosting: JobPosting,
```